### PR TITLE
Kyrylo

### PR DESCRIPTION
**Pull request by xioru**
Scooter station changes

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -21,9 +21,9 @@ services:
     container_name: scooter_db
     env_file: 
       - ./configs/.env
-    volumes:
+    #volumes:
```
Comment (DmytroTHR) [inline docker-compose.yml:24]: no need to change this file, I think

File: go.mod [hunk 2]
```diff
@@ -5,16 +5,13 @@ module Dp218Go
 go 1.15
 
 require (
```
Comment (DmytroTHR) [inline go.mod:7]: no need to change this file, I think

File: go.sum [hunk 3]
```diff
@@ -290,7 +290,6 @@ github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSs
 github.com/denisenkom/go-mssqldb v0.10.0/go.mod h1:xbL0rPBG9cCiLr28tMa8zpbdarY27NDyej4t/EjAShU=
```
Comment (DmytroTHR) [inline go.sum:290]: no need to change this file, I think

File: repositories/postgres/user.go [hunk 4]
```diff
@@ -33,7 +33,7 @@ func (urdb *UserRepoDB) GetAllUsers() (*models.UserList, error) {
 	for rows.Next() {
```
Comment (DmytroTHR) [inline repositories/postgres/user.go:33]: no need to change this file, I think

**Review by DmytroTHR — CHANGES_REQUESTED** [review]
There shouldn't be files marked with my recent comments, I think. Try to pull the last develop once again and merge it with your branch (replacing marked files in your branch to ones 
 from develop).

**Review by IntelliJIN — CHANGES_REQUESTED** [review]
Please add comments to structures and methods/functions.

**Pull request closed without merging**